packages/deepsignal/react/test/index.test.tsx

```diff
@@ -0,0 +1,55 @@
+// @ts-ignore
+globalThis.IS_REACT_ACT_ENVIRONMENT = true;
+
+// @ts-ignore
```

Comment:
It would be great to add a description for these `@ts-ignore` comments, as recommended in the [TypeScript docs](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-6.html#:~:text=It%20is%20recommended%20practice%20to%20have%20the%20remainder%20of%20the%20comment%20following%20%40ts%2Dignore%20explain%20which%20error%20is%20being%20suppressed.). E.g.,

```ts
if (false) {
  // @ts-ignore: Unreachable code error
  console.log("hello");
}
```

Reply:
To be honest, I don't know the exact reason because the configuration was copied from the original preact signals repo 🤷‍♂️😄